src/main/java/com/example/notificationservice/service/SqsListenerService.java

```diff
@@ -0,0 +1,76 @@
+package com.example.notificationservice.service;
```

Comment:
please make sure you respect the standard package naming convention pflarache.smartlib.notifications.service
{client}.{app}.{microService}.{layer}